src/main/java/org/lsc/Hooks.java

```diff
@@ -58,62 +58,33 @@
  */
 public class Hooks {
 
-	static final Logger LOGGER = LoggerFactory.getLogger(AbstractSynchronize.class);
+	static final Logger LOGGER = LoggerFactory.getLogger(Hooks.class);
+
+	private static final ObjectMapper Mapper = new ObjectMapper();
 	/**
 	 * Method calling a postSyncHook if necessary
 	 *
 	 * return nothing
 	 */
-	public final static void postSyncHook(final String hook, final String outputFormat, final LscModifications lm) {
+	public final void postSyncHook(final Optional<String> hook, final OutputFormat outputFormat, final LscModifications lm) {
 
-		if( hook != null && ! hook.equals("") )
+		if( hook != null && hook.isPresent() )
```

Comment:
It looks better.
The principle of an optional is to not be null, so you can remove the null check.

Also if instead of a String operationType you continue using LscModificationType, the code can be very simplified by:
```
hook.ifPresent(h -> callHook(lm.getOperation(), h, lm.getMainIdentifier(), outputFormat, lm)
```